### ISSUE TITLE
Displaying title and then removing title in Success Message notification.

### ISSUE THREAD

**Issue by paaljoachim** [wpninjas/ninja-forms#378]
One issue that I noticed...

Displaying the title of the contact form on the contact page how would I go about removing the title on the Success message notification "page"?

As the contact page can have a text that says "send me en email" or something similar to that. Filling out the form and then sending it in there is no need to have the same title in the top. It would be better to have the title removed from the success page. This means creating a check box in the Success message notification where one can remove the contact form title. To only show the text added into the Success message content area. AND perhaps also a check box that says remove content from page if information has been added to the contact page in the content creation area.

Bottom line:
- Success message notification check box for removing form title and another for hiding page content.
So one content for the regular contact form page and another for the success message notification.

**Comment by kstover** [comment]
I think that this should be taken care of now; there were some bugs in how we handled adding and removing rich text editors via JS.
